net.sourceforge.roccat.roccat-tools.yml

```
@@ -0,0 +1,73 @@
+app-id: net.sourceforge.roccat.roccat-tools
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+rename-icon: roccat
+
+cleanup:
+  - /include
+  - /share/man
+  - /lib/pkgconfig
+
+modules:
+  - shared-modules/lua5.4/lua-5.4.json
+  - shared-modules/libcanberra/libcanberra.json
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/dbus-glib/dbus-glib.json
+  - shared-modules/gudev/gudev.json
+
+  - name: libnotify
+    buildsystem: meson
+    config-opts:
+      - -Dman=false
+      - -Dgtk_doc=false
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOE/libnotify/-/archive/0.8.2/libnotify-0.8.2.tar.gz
+        sha256: 84d6bbcd633ff10eed25226730c52220cd41e362bb183513e5b6a5d1b3c0c12f
+
+  - name: libgaminggear
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_C_FLAGS=-isystem /usr/include/harfbuzz
+      - -DINSTALL_PKGCONFIGDIR=/app/lib/pkgconfig
+      - -DINSTALL_CMAKE_MODULESDIR=/app/lib/cmake
+      - -DINSTALL_LIBDIR=/app/lib
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/libgaminggear/libgaminggear-0.15.1.tar.bz2
+        sha256: ffbd59c62e3107b09ec99f1e5147529c61931987abc7f86a140449b46388c549
+
+  - name: roccat-tools
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_C_FLAGS=-isystem /usr/include/harfbuzz
+      - -DCMAKE_MODULE_PATH=/app/lib/cmake
+      - -DINSTALL_LIBDIR=/app/lib
+      - -DUDEVDIR=/app/lib/udev
+      - -DLIBDIR=/app/lib
+      - -DWITH_LUA=5.4
+    post-install:
+      # Install metainfo
+      - install -Dm644 net.sourceforge.roccat.roccat-tools.metainfo.xml /app/share/metainfo/net.sourceforge.roccat.roccat-tools.metainfo.xml
+      # Rename desktop files
+      - cd /app/share/applications/ && for desktop_filename in *; do mv "$desktop_filename" "net.sourceforge.roccat.roccat-tools.$desktop_filename"; done;
```

```suggestion
      - cd /app/share/applications/ && for desktop_filename in *; do mv "$desktop_filename" "net.sourceforge.roccat.roccat_tools.$desktop_filename"; done;
```

It's subtle but it's a side effect of having a `-` in the package name.

---

I'm not sure to understand here ? If I change this, the desktop files (one for each device) will not have proper naming and will not be launchable from my desktop ?

---

How about you trust the reviewer. Or better, just try it.

---

I expressed myself wrongly: I have already tried, and can confirm these desktop files does not appear on my desktop anymore when I change this line.

Maybe I missed something ?

---

that it is a bug then.

`net.sourceforge.roccat.roccat-tools.roccatarvoconfig` is an invalid app-id. Everywhere else `-` is replaced by `_`.  Like:

> `Exporting net.sourceforge.roccat.roccat_tools.Debug to repo`

If it work that way we can leave it that way.